seed/views/00600_individual_court_reporter_overview.js

```diff
@@ -1,10 +1,9 @@
 exports.seed = function (knex, Promise) {
-  var sql = `CREATE VIEW app.individual_court_reporter_overview
+  var view = `CREATE VIEW app.individual_court_reporter_overview
       WITH SCHEMABINDING 
       AS 
       SELECT
-        wo.id AS id 
```

Comment:
Why remove the `id` field? The where clause in the web app uses this field

Reply:
Didn't mean to add this file, my bad.